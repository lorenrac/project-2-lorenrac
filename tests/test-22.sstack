// Test the contains operator
VAR magic = "abra cadabra"
PRINT "Scanning '" + magic + "'..."
IF (magic ? 'ab') PRINT " ...has 'ab'"
IF (magic ? 'bc') PRINT " ...has 'bc'"
IF (magic ? 'ca') PRINT " ...has 'ca'"
