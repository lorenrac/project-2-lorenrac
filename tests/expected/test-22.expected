Scanning 'abra cadabra'...
 ...has 'ab'
 ...has 'ca'
